第2話追加

sekiten42 opened the pull request:
2話目追加したからみてー

sekiten42 committed:
Added files via upload

n話の設定ファイル
「旋盤で縦ロール」

sekiten42 committed:
Update characters.adoc

仮タイトル「旋盤で縦ロール」の登場人物2名とその設定を追加。

sekiten42 committed:
Update characters.adoc

仮タイトル「旋盤で縦ロール」の追加登場人物名について
ふりがな表記が間違っていたのでまわりに合わせた。

sekiten42 committed:
Added files via upload

とりあえずのファーストアップロード
仮タイトル「旋盤で縦ロール」

sekiten42 committed:
Update 002.adoc

美弥乃が作業着をきる設定を削除
追記分に関する設定を追加

sekiten42 committed:
Update 002.adoc

表記方法間違えた

sekiten42 committed:
Update 002-工学院・機械系_美弥乃.txt

追記と概略の削除
著者名表記法を前話と統一

まだ全体読み直してないから話が通ってるかわからんけどとりあえずcommit

sekiten42 committed:
Update 002-工学院・機械系_美弥乃.txt

修正加えたけどちょっとがばがば感

sekiten42 committed:
Update 002-工学院・機械系_美弥乃.txt

気になった個所修正

Commit:
Merge branch 'master' of https://github.com/l1048576/tk-jogakuin into l1048576-master

Conflicts:
	settings/characters.adoc

Commit:
場所の追加

lo48576 commented on main/002-工学院・機械系_美弥乃.txt:
```diff
@@ -0,0 +1,147 @@
+　「どうかしら？」
+　「え、私なんかがこんな髪型をしていいのでしょうか...」
+　「いいのよ。あなたはこの女学院に入学した一人の学生なのだから。
+　　女学院生であることに誇りを持ちなさいな」
```
改行不要な気がします

l1048576 commented:
# 文法について
## エラー
- 全角ダブルクォートの「”」は閉じに使うもので、開きは「“」を使う
## エラーではないが
- 三点リーダは「……」のように、「…」を偶数個並べるのが原則。ピリオドを並べない、
- 鍵括弧の末尾に句点(。)は普通置かない
- 「///」は漫符を文字化したもので、普通のスタイルの小説では使わない(文章で表現する)
- 疑問符「？」の直後に文章が続くなら、感嘆符と同様、後ろに全角空白を置く

べつにエラーでないものは常識というだけで必須ではないので、こだわりがあるなら修正の必要はないです。

# その他
他の個別事項についてはコミットにコメントします

lo48576 commented on main/002-工学院・機械系_美弥乃.txt:
```diff
@@ -0,0 +1,147 @@
+　しきりに周りで同じような人がいないかと探していたとき、突然足元が不安定になり転んでしまった。
+　幸いにも怪我はしなかったが、困ったことにドレスが汚れてしまった。
+
+　(このままじゃ歓迎パーティに出られない...。どこかで身だしなみを整えなくては...。)
```
「このままじゃ」と「整えなくては」って、口調がぶれている気がします。「このままでは」にするか、「整えなくちゃ」とか「整えないと」にするのが自然では？

l1048576 commented:
# 文法について
## エラー
- 全角ダブルクォートの「”」は閉じに使うもので、開きは「“」を使う
## エラーではないが
- 三点リーダは「……」のように、「…」を偶数個並べるのが原則。ピリオドを並べない、
- 鍵括弧の末尾に句点(。)は普通置かない
- 「///」は漫符を文字化したもので、普通のスタイルの小説では使わない(文章で表現する)
- 疑問符「？」の直後に文章が続くなら、感嘆符と同様、後ろに全角空白を置く

べつにエラーでないものは常識というだけで必須ではないので、こだわりがあるなら修正の必要はないです。

# その他
他の個別事項についてはコミットにコメントします

l1048576 commented:
ごめんｽﾏﾋｮだったからボタン押し間違えた

lo48576 commented on main/002-工学院・機械系_美弥乃.txt:
```diff
@@ -0,0 +1,147 @@
+　
+　しばらくして大変なことを思い出してしまった。
+　お姉様の名前を聞き忘れたのだ！
+　でも今から戻ってもお姉様がまだいるかわからない上に、パーティーに遅れてしまう。
```
114行目で地の文で「おっしゃった」と敬語を使っているから、ここでも「いらっしゃるかわからない」とするべきでは

Commit:
002-工学院・機械系_美弥乃.txt：
・指摘された点の修正と気になった個所の修正
・場所の名称変更(実在するものと関係ないとはいえ少し気になった)
場所.adoc：
・本文中での呼称変更に伴う修正

sekiten42 commented:
修正したにゃ！
